ironfish/src/rpc/server.ts

```diff
@@ -73,4 +76,21 @@ export class RpcServer {
       this._startPromise = promise
     }
   }
+
+  /** Authenticate the RPC request */
+  authenticate(requestAuthToken: string | undefined): boolean {
+    if (requestAuthToken) {
+      this.logger.debug(`Missing Auth token in RPC request.`)
+      return false
+    }
+
+    const rpcAuthToken = this.node.internal.get('rpcAuthToken')
+
+    if (rpcAuthToken || rpcAuthToken === '') {
```

Comment:
This will always go into this branch because any non empty string is truthy

Reply:
It should be ```if(!rpcAuthToken)``` updated